### PR TITLE
Added bmp login in integration test

### PR DESCRIPTION
**Pull request by zhanggbj**
1.Added bmp login in integration test
2.Added bmp login in bmp status test

[#118158419](https://www.pivotaltracker.com/story/show/118158419)

### CODE REVIEW

File: integration/bmp/status_cmd_test.go [hunk 1]
```diff
@@ -12,6 +12,11 @@ import (
 var _ = Describe("`$bmp status` integration tests", func() {
 	var session *Session
 
+	BeforeEach(func() {
+		session = RunBmpLogin()
+		Expect(session.ExitCode()).To(Equal(0))
+	})
+
 	Describe("$bmp status", func() {
```
Comment (maximilien) [inline integration/bmp/status_cmd_test.go:23]: should'nt that be `$bmp login` in description?

File: integration/bmp/status_cmd_test.go [hunk 2]
```diff
@@ -12,6 +12,11 @@ import (
 var _ = Describe("`$bmp status` integration tests", func() {
```
Comment (maximilien) [inline integration/bmp/status_cmd_test.go:12]: should'nt that be `$bmp login` in description?

**Comment by maximilien** [comment]
Please fix the descriptions and I'll merge this.

Also, a great additional tests would be to verify that the user's config file (in `~/.bmp_config`) has the `Username` and `Password` if the `$bmp login username password` suceeded.

File: integration/bmp/status_cmd_test.go (hunk 2 above)
Comment (maximilien) [inline integration/bmp/status_cmd_test.go:12]: Maybe it is bmp status?

**Comment by maximilien** [comment]
So you will need to rebase with previous PR merge.

File: integration/bmp/status_cmd_test.go (hunk 2 above)
Comment (zhanggbj) [inline integration/bmp/status_cmd_test.go:12]: Yes. This is the bmp status integration test. bmp login is executed in BeforeEach.

**Comment by maximilien** [comment]
Nice, merging 👍

**Pull request merged**